Public/Select-ReleaseNotes.ps1

```
@@ -190,7 +190,7 @@ function Select-ReleaseNotes {
     $HeaderRegex = '^#+\s*(?<header>.+):?\s*$'
     $DateRegex = '^#+\s*.*(?<date>\d\d\d\d.\d\d.\d\d)'
     $StraplineStartRegex = '^#+\s*Strapline\s*$'
-    $StraplineRegex = '^\s*(?<priority>[0-9]+)\s*-?\s*(?<feature>[^\.]*)\s*$'
+    $StraplineRegex = '^\s*(?<priority>[0-9]+)\s*-\s*(?<feature>.*)\s*$'
 
```

This is a breaking change. Are we sure that builds with release notes that don't currently contain the mandatory dash won't silently fail? i.e. do builds blow up if the new format isn't satisfied, or will the strapline simply be absent from those builds?